pyproject.toml

```diff
@@ -68,6 +68,7 @@ disable = [
     "format",
 #    "abstract-class-little-used",
     "abstract-method",
+    "consider-alternative-union-syntax",
```

Comment:
When Python 3.10 is the minimum version (or pydantic is updated to support the new union syntax with Python 3.9) we can remove this disable and change to the new syntax.

Reply:
Maybe add this as a comment in the code for future reference?